src/test/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCallerTest.scala

```diff
@@ -383,6 +383,20 @@ class VanillaUmiConsensusCallerTest extends UnitSpec with OptionValues {
     consensus.getFloatAttribute(ConsensusTags.PerRead.RawReadErrorRate) shouldBe 0.toFloat
   }
 
+  it should "calculate the # of errors relative to the most likely consensus call, even when the final call is an N" in {
```

Comment:
Moved this test to the section it belonged to.  It wasn't testing `filterToMostCommonAlignment`